doc/cabal-package.rst

```diff
@@ -1037,39 +1037,36 @@ The command writes the selected version for all dependencies to the
 the dependency versions specified in it.
 
 Generating dependency version bounds
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+""""""""""""""""""""""""""""""""""""
 
 Cabal also has the ability to suggest dependency version bounds that
-conform to the `Package Versioning Policy`_, which is
+conform to `Package Versioning Policy`_, which is
 a recommended versioning system for publicly released Cabal packages.
 This is done by running the ``gen-bounds`` command:
 
 .. code-block:: console
 
     $ cabal gen-bounds
 
-For example, given the following dependencies without bounds specified in
+For example, given the following dependencies specified in
 :pkg-field:`build-depends`:
 
 ::
 
     build-depends:
-      base,
-      mtl,
-      transformers,
+      foo == 0.5.2
+      bar == 1.1
 
-``gen-bounds`` might suggest changing them to the following:
+``gen-bounds`` will suggest changing them to the following:
 
 ::
 
     build-depends:
-      base          >= 4.15.0 && < 4.16,
-      mtl           >= 2.2.2 && < 2.3,
-      transformers  >= 0.5.6 && < 0.6,
-
+      foo >= 0.5.2 && < 0.6
+      bar >= 1.1 && < 1.2
```

Comment:
The semantics of this section have changed, and are now less informative than before. Please revert. The point of this section is to show that bounds-less dependencies in the presence of a `cabal gen-bounds` run produces a set of bounds for those deps.